x-pack/plugins/transform/public/app/sections/transform_management/components/action_delete/delete_action_name.tsx

```diff
@@ -24,45 +26,71 @@ export const deleteActionNameText = i18n.translate(
 );
 
 const transformCanNotBeDeleted = (i: TransformListRow) =>
+  i.stats &&
   !([TRANSFORM_STATE.STOPPED, TRANSFORM_STATE.FAILED] as TransformState[]).includes(i.stats.state);
 
 export const isDeleteActionDisabled = (items: TransformListRow[], forceDisable: boolean) => {
   const disabled = items.some(transformCanNotBeDeleted);
-  return forceDisable === true || disabled;
+  const hasNoStats = items.some((i: TransformListRow) => !isTransformListRowWithStats(i));
+
+  return forceDisable === true || disabled || hasNoStats;
 };
 
 export interface DeleteActionNameProps {
+  items: TransformListRow[];
   canDeleteTransform: boolean;
   disabled: boolean;
   isBulkAction: boolean;
+  forceDisable: boolean;
 }
 
+export const getDeleteActionDisabledMessage = ({
+  items,
+  canDeleteTransform,
+  forceDisable,
+}: {
+  items: TransformListRow[];
+  canDeleteTransform: TransformCapabilities['canDeleteTransform'];
+  forceDisable: boolean;
+}) => {
+  const isBulkAction = items.length > 1;
+
+  // Disable start for transforms if stats does not exist
```

Comment:
Nit - should be `Disable delete ...`

Reply:
Removed here [`82de19d` (#166320)](https://github.com/elastic/kibana/pull/166320/commits/82de19d62bea414640929fd4b106c105a51e1a53)